test/unit/cpu/test-forall.cxx

```diff
@@ -124,3 +124,18 @@ using CilkTypes = ::testing::Types<
 
 INSTANTIATE_TYPED_TEST_CASE_P(Cilk, ForallTest, CilkTypes);
 #endif
+
+#if defined(RAJA_ENABLE_AGENCY)
```

Comment:
This uses GTest templating to reduce the amount of code it takes to setup each test